tests/test-definitions.sh

```diff
@@ -43,7 +43,7 @@ function verify_ib_device_status {
     check_exit_code "IB device state: LinkUp" "IB link not up"
 
     # verify ifconfig
-    ifconfig | grep "ib0:\|ibP"
+    ifconfig | grep "ib[0-9]:\|ibP"
```

Comment:
0-9? I don't think there's a SKU with more than 8 IB devices

Reply:
It just made more sense to me to "allow all digits" rather than saying 0-7.  Though if there's a more succinct way to do this ('d'?) then I would use that. 

When I asked yesterday I was trying to determine if I should allow for multiple digits.

Reply:
Totally forgot that 0-9 is how to grep for all digits, I think this make sense.